plugins/modules/make.py

```diff
@@ -83,7 +83,36 @@
     file: /some-project/Makefile
 '''
 
-RETURN = r'''# '''
+RETURN = r'''
+chdir:
+  description:
+    - Change to this directory before running make.
+  type: str
+  returned: always
```

Comment:
This and the other return values are only returned on success:
```suggestion
  returned: success
```